cmd/serve/s3/signature/signature-errors.go

```diff
@@ -0,0 +1,146 @@
+package signature
```

Comment:
There seems to be a lot of boilerplate that should be supplied by gofakes in here?

Reply:
Changed. Moved code to gofakes3